Classes/Issues/Comments/Reactions/IssueReactionCell.swift

```diff
@@ -35,7 +35,7 @@ final class IssueReactionCell: UICollectionViewCell {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
-        accessibilityTraits |= UIAccessibilityTraitButton
+        accessibilityTraits.insert(.button)
```

Comment:
None of these were fixed by the migrator 😭

Reply:
@BasThomas Highly recommend opening an issue about that. When I had issues with the migrator I opened an issue and they fixed it with the next release of Swift.

Reply:
Thanks, that's a good idea! Found https://bugs.swift.org/browse/SR-8181.